autoload/go/guru.vim

```diff
@@ -364,6 +363,12 @@ function! go#guru#SameIds(selected)
     let pos = split(item, ':')
     call matchaddpos('goSameId', [[str2nr(pos[-2]), str2nr(pos[-1]), str2nr(poslen)]])
   endfor
+
+  if get(g:, "go_auto_sameids", 0)
+    " re-apply SameIds at the current cursor position at the time the buffer
+    " is redisplayed: e.g. :edit, :GoRename, etc.
+    autocmd BufWinEnter <buffer> nested call go#guru#SameIds(-1)
```

Comment:
So this is called because we call `:e` inside `:GoRename`, which loads the buffer and thus it triggers this line right ? I wonder if there is a case that doesn't remove the trigger below.